package.json

```diff
@@ -51,6 +52,8 @@
     "rules": {
       "no-console": 1,
       "no-empty": 0,
+      "semi": 2,
+      "keyword-spacing": 2,
```

Comment:
@developit I took the liberty of adding those rules to eslint - please check if they're ok with you.
Semicolons are enforced (they were placed almost everywhere and not enforced) & spaces should be placed after keyword (no changes to files).